app/components/record_details/view.rb

```diff
@@ -106,7 +111,9 @@ def record_created_row
 
     def trainee_start_date_row
       start_date = RecordDetails::TraineeStartDate.new(trainee)
-
+      if start_date.course_empty? && trainee.hesa_record?
+        start_date = t("components.confirmation.not_provided_from_hesa_update")
```

Comment:
I don't think this quite works.

If we set `start_date` to a "not provided" string inside this if-statement, then the line underneath will try to call `course_empy?` on a string, which will error.

Reply:
We should probably add a test for this case.

Reply:
Test added and start_date fixed